### DISCUSSION TITLE
[Feature] Download multiple single pictures (instead of a .ZIP)

### DISCUSSION THREAD

**Discussion by TaktischerSpeck** [immich-app/immich#10879] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I love Immich, but sometimes I want to share multiple pictures with non Immich users.
“My” Problem is that not “techie” users don't know how to extract a .ZIP (at least on the phone).

Is it possible to create an option to download multiple single pictures so that you can just move them to the camera roll?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by skylatian** [comment]
I would like to bump this! I try to send photos to friends, and extracting a ZIP file isn't familiar to most of them - and even to those who are tech-savvy, extracting a zip file and saving it to photos (especially on iOS) is rather tedious.
